packaging/ubuntu-16.04/snapd.maintscript

```diff
@@ -2,4 +2,4 @@
 # we used to ship a custom grub config that is no longer needed
 rm_conffile /etc/grub.d/09_snappy 1.7.3ubuntu1
 rm_conffile /etc/ld.so.conf.d/snappy.conf 2.0.7~
-rm_conffile /etc/apparmor.d/usr.lib.snapd.snap-confine 2.23.6~ snap-confine
+rm_conffile /etc/apparmor.d/usr.lib.snapd.snap-confine 2.23.6~
```

Comment:
What is the semantics of this change? I understand it was referring to the snap-confine package before and now it no longer does so. Why?

Reply:
It was changed to snap-confine erroneously by me in a previous version of this that was never uploaded to the ubuntu archive.

Reply:
package name specification is redundant is automatically derived, and should normally not be used at all.